From 2c4921501e40397b0dcbff443dc2765e14610685 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Mon, 19 Sep 2016 09:16:09 +0300
Subject: [PATCH 43/51] regulator: userspace-consumer: add MODULE_ALIAS macro

Enable userspace-consumer module initiation.
Without the macro the module can only be initiated by the
modprobe/insmod commands.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 drivers/regulator/userspace-consumer.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/regulator/userspace-consumer.c b/drivers/regulator/userspace-consumer.c
index 765acc1..25d4d30 100644
--- a/drivers/regulator/userspace-consumer.c
+++ b/drivers/regulator/userspace-consumer.c
@@ -184,3 +184,4 @@ static int regulator_userspace_consumer_remove(struct platform_device *pdev)
 MODULE_AUTHOR("Mike Rapoport <mike@compulab.co.il>");
 MODULE_DESCRIPTION("Userspace consumer for voltage and current regulators");
 MODULE_LICENSE("GPL");
+MODULE_ALIAS("platform:reg-userspace-consumer");
-- 
1.9.1

